.github/workflows/v1-ci.yml

```diff
@@ -11,32 +11,36 @@ jobs:
     name: Build
     runs-on: ubuntu-latest
     env:
-      GOPATH: ${{ github.workspace }}
       WORKDIR: src/github.com/${{ github.repository }}
 
     strategy:
       matrix:
-        go: [ '1.15', '1.14', '1.13', '1.12' ]
+        go: [ '1.21', '1.20', '1.19', '1.18', '1.17' ]
```

Comment:
Any particular reason to start at 1.17?

Reply:
We can include `1.16` but we can't use older than that because `io/ioutil` has been deprecated from version `1.16`.

Reply:
Turns out `golangci-lint` fails if we include `1.16`.

Reply:
okie-doke